src/Container/Definition/Definition.php

```diff
@@ -276,6 +284,22 @@ protected function resolveCallable(callable $concrete): mixed
      */
     protected function resolveClass(string $concrete): object
     {
+//        $reflector = new ReflectionClass($concrete);
+//        $construct = $reflector->getConstructor();
+//        $params = $construct->getParameters();
+//        if (count($params) !== count($this->arguments)) {
+//            // This indicates, that certain arguments need to be auto wired
+//            $tmpArgs = $this->arguments;
+//            $this->arguments = [];
+//            foreach($params as $param) {
+//                if (!array_key_exists($param->getName(), $tmpArgs)) {
+//                    $this->addArgument($param->getType()->getName(), $param->getName());
+//                } else {
+//                    $this->addArgument($tmpArgs[$param->getName()], $param->getName());
+//                }
+//            }
+//        }
```

Comment:
This was my first attempt to make partial definitions work with autowiring but I get an infinite loop doing this.